### PR TITLE
Sent inquiry

### PR DESCRIPTION
**Pull request by anipetrov**
The type of this PR is an enhancement of an existing event, `sent_artwork_inquiry`
 -->

### PR Checklist (tick all before merging)

- [x] If I've added a new file to the tree I've exported it from the common `index.ts`
- [x] I've added comments with examples for any new interfaces and ensured that they're in the docs
- [x] No platform-specific terminology has been used outside of `click` and `tap` (platform is inferred by the DB storing events)

### CODE REVIEW

File: src/Schema/Events/Inquiries.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+/**
+ * The type of product associated with an inquiry request
+ */
+export type InquiryProduct = {
+  price: number
+  product_id: string
+  quantity: 1
+}[]
```
Comment (dzucconi) [inline src/Schema/Events/Inquiries.ts:46]: You'd type this without the array; then include the array form in the `products` field above (`products: InquiryProduct[]`)

**Comment by anipetrov** [comment]
gah looks like i was on an old branch, will edit on monday

File: src/Schema/Events/Inquiries.ts (hunk 1 above)
Comment (anipetrov) [reply src/Schema/Events/Inquiries.ts:46]: thanks!

**Comment by dzucconi** [comment]
Apparently the product data is for Criteo? https://github.com/artsy/force/pull/4041

(Any ideas for the `6000` default? https://github.com/artsy/force/blob/938ae6d7ac57c93141052dd05821d15850dbab63/src/desktop/components/inquiry_questionnaire/analytics/events.ts#L168)

File: src/Schema/Events/Inquiries.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ *  @example
+ *  ```
+ *  {
+ *    action: "sentArtworkInquiry",
+ *    artwork_id: "582b1bc09c18db3b0c000999",
+ *    artwork_slug: "philippe-halsman-alfred-hitchcock-during-the-filming-of-the-birds",
+ *    inquiry_id: "614e3063ef0b81000bc4f45a",
+ *    products: [
+        {
+          price: 6000,
```
Comment (dzucconi) [inline src/Schema/Events/Inquiries.ts:22]: I'm pretty sure this is a pre-formatted string most of the time? The default just happens to be 6000 for some reason?

**Comment by abhitip** [comment]
Oh interesting, thanks for finding that @dzucconi ! I think we can remove the product id, right @anipetrov ? I feel like most marketing integrations I'm seeing these days will let us map the artwork id to their internal definition, rather than forcing us to name it how they'd like. Lmk what you think!

File: src/Schema/Events/Inquiries.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  artwork_slug: string
+  inquiry_id: string
+  products: InquiryProduct
+}
+
+/**
+ * The type of product associated with an inquiry request
+ */
+export type InquiryProduct = {
+  price: number
```
Comment (dzucconi) [inline src/Schema/Events/Inquiries.ts:43]: This is likely a string — if we can check the data to find out what it's receiving that'd be great.

**Comment by dzucconi** [comment]
I think `e305932` is causing this to fail in master. I'm going to revert then just rebase this.

**Pull request merged**